core/commands/id.go

```diff
@@ -19,8 +19,9 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
-const offlineIdErrorMessage = `ID command fails when run without daemon, we are working to fix this.
-In the meantime, please run the daemon if you want to use 'ipfs id':
+const offlineIdErrorMessage = `'ipfs id' currently cannot query information on remote
+peers without a running daemon; we are working to fix this.
+In the meantime, please run the daemon if you want to use 'ipfs id' in that way:
```

Comment:
"In the meantime, if you want to query remote peers using 'ipfs id', please run the daemon:"